test/python/transpiler/test_pulse_gate_pass.py

```diff
@@ -159,15 +159,14 @@ def test_transpile_with_custom_basis_gate_in_target(self):
         qc.sx(1)
         qc.measure_all()
 
-        transpiled_qc = transpile(qc, initial_layout=[0, 1], target=target)
-
         ref_calibration = {
             "sx": {
                 ((0,), ()): self.custom_sx_q0,
                 ((1,), ()): self.custom_sx_q1,
             }
         }
         with self.assertWarns(DeprecationWarning):
+            transpiled_qc = transpile(qc, initial_layout=[0, 1], target=target)
```

Comment:
This should go back outside of the `self.assertWarns` block (to line 162); unless I'm missing here something, we are not expecting `transpile` to emit deprecation warnings here